service/routes.py

```diff
@@ -31,7 +31,23 @@ def index():
 
 @app.route("/recommendations", methods=["GET"])
 def list_recommendations():
-    pass
+    products = Recommendation.all()
```

Comment:
`products` -> `recommendations`

Reply:
I think this is value name, what we get will be products

Reply:
In the database we have recommendations, not products.
Out Recommendation model have attributes:
`id` : recommendation id
`src_product_id` :  source product id
`rec_product_id` : recommened product id
`type`: recommendation type

Reply:
ok, but they still products, anyway, i will change it